### PR TITLE
SO-4659: Map compare to consider active mappings only

### PR DESCRIPTION
**Pull request by molnarlaura**

### CODE REVIEW

File: core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptMapMappingSearchRequestBuilder.java [hunk 1]
```diff
@@ -67,6 +67,10 @@ public ConceptMapMappingSearchRequestBuilder filterByComponentIds(Iterable<Strin
 		return addOption(OptionKey.COMPONENT, componentIds);
 	}
 	
+	public ConceptMapMappingSearchRequestBuilder filterByActive(boolean active) {
```
Comment (cmark) [inline core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptMapMappingSearchRequestBuilder.java]: Use boxed `Boolean` instead.

**Review by cmark — APPROVED** [review]
LGTM!

File: core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptMapMappingSearchRequestBuilder.java (hunk 1 above)
Comment (molnarlaura) [reply core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptMapMappingSearchRequestBuilder.java]: Done

**Review by cmark — APPROVED** [review]
LGTM!

**Pull request merged**